message.go

```diff
@@ -36,9 +36,73 @@ type Message struct {
 	Message    string    `json:"message"`
 	Suggestion string    `json:"suggestion,omitempty"`
 	Time       time.Time `json:"time,omitempty"`
+	Origin 	   string	 `json:"origin" example:"from:api/from:gdc"`
 }
 
 func sendDiscordMessage(s *discordgo.Session, channelID string, msg Message) error {
-	_, err := s.ChannelMessageSend(channelID, fmt.Sprintf("**TYPE**\n%s\nMessage:%s\nSuggestion:%s", msg.Type, msg.Message, msg.Suggestion))
-	return err
+
+	embed_full := &discordgo.MessageEmbed{
+		Author:      &discordgo.MessageEmbedAuthor{Name: "Adomate Bot"},
+    	Color:       0x00ff00, // Green - should change later based on message 
+		Description: "This is a message from the api bot",
+		Fields: []*discordgo.MessageEmbedField{
+			&discordgo.MessageEmbedField{
```

Comment:
I think this is a redundant declaration, you can just do 
`
{
  Name:xxx
  Value:xx
  Inline:xxx
}`

Since you declared the array of type discordgo.MessageEmbedField, it knows each object is going to be of that type, no need to specify again.